2025-07-19 23:12:51,720 - dbbase.py - ERROR - (1115, "Unknown character set: 'utf8mb4'")
